modules/govuk/templates/ckan/pycsw.cfg.erb

```diff
@@ -0,0 +1,61 @@
+[server]
+home=/var/ckan
+url=http://localhost:<%= @pycsw_port %>/csw
+mimetype=application/xml; charset=UTF-8
+encoding=UTF-8
+language=en-GB
+maxrecords=10
+loglevel=INFO
+logfile=/var/log/ckan/pycsw.log
+#ogc_schemas_base=http://foo
+#federatedcatalogues=http://catalog.data.gov/csw
+#pretty_print=true
+gzip_compresslevel=9
+#domainquerytype=range
+#domaincounts=true
+#spatial_ranking=true
+profiles=apiso
+
+[manager]
+transactions=false
+allowed_ips=127.0.0.1
+#csw_harvest_pagesize=10
+
+[metadata:main]
+identification_title=pycsw Geospatial Catalogue
+identification_abstract=pycsw is an OGC CSW server implementation written in Python
+identification_keywords=catalogue,discovery,metadata
+identification_keywords_type=theme
+identification_fees=None
+identification_accessconstraints=None
+provider_name=Organization Name
```

Comment:
Should any of this (or the fields below) be filled in?

Reply:
To be honest I don't really know, I'm not familiar enough with CSW or where they would be exposed. I'll have a look around and see what we might want to set them to.

Reply:
I think that's fine. Providing we're aware that there is this configuration here.